### ISSUE TITLE
Analytics Feature Request/Bug Report

### ISSUE THREAD

**Issue by zlaw** [department-of-veterans-affairs/va.gov-team#53962]
# Analytics Feature Request/Bug Report

## What this form is for

Adding the https://www.va.gov/education/after-you-apply/ landing page to Domo.

Platform Analytics & Insights will use the information you provide to help improve our products and services.

## Instructions

Please fill out the following sections:

- [ ] Ticket Description
- [ ] Request/Report Description
- [ ] Additional Comments

## Ticket Description

Who is this request for?

- My name is Zachary Law, a Product Manager with VA Notfiy.

- Is this a feature request or a bug?

The is a feature request.

## Request/Report Description

We are trying to track UTM analytics in Domo for a new landing page: https://www.va.gov/education/after-you-apply/ so we can track our UTM parameter metrics ?utm_source=VANotify&utm_medium=email&utm_campaign=22-5490-confirmation

For a new feature request:

We want this to track notification KPI's over time

## Acceptance Criteria

- [ ] Page added to Landing pages - VANotify users chart
- [ ] UTM parameter tracked by campaign name

For features:

- [ ] Groomed & backlogged with notes

For bugs:

- [ ] Document issue & fixes
- [ ] Report findings with team & requester

## Definition of Done

- [ ] All AC completed

**Comment by zlaw** [comment]
Hi Michelle,

That's correct!  The campaign code is 22-5490-confirmation.

Also, would it be possible to also remove the "VA Notify - Test" service from Domo?  

Thanks a lot,
Zack

**Comment by dorseypa1** [comment]
Hi Amanda,

Just a heads up that I still haven't received the automated email

**Comment by zlaw** [comment]
Hi @michelle-dooley,

Just making sure you saw my comment since i didn't @ your name.

Thanks!
Zack

**Comment by michelle-dooley** [comment]
@zlaw yes I saw your note.  @dorseypa1 is working on your request, he will reach out with any questions.

**Comment by dorseypa1** [comment]
The new page is now flowing into Domo.  
![image](https://user-images.githubusercontent.com/117392065/223762900-b598f114-ac9d-498c-85e5-cee47c3a0d63.png)

**Comment by zlaw** [comment]
Thanks, @michelle-dooley and @dorseypa1 ! I was also wondering if you could remove the "VA Notify - Test" service from domo as well... let me know!

**Comment by dorseypa1** [comment]
Moving on to the "VA Notify - Test" portion of the request next.  I'll give you an update once I've had a chance to dig in.

**Comment by zlaw** [comment]
thanks, @dorseypa1 !  Much appreciated!

**Comment by dorseypa1** [comment]
This is what I see in Domo when I search for 'VA Notify - Test',  2 cards and 1 Dashboard.  Are these the items you want removed? 
![image](https://user-images.githubusercontent.com/117392065/223794542-9014ef9d-3144-40fa-ae62-7d69c1936f73.png)

**Comment by zlaw** [comment]
Hey @dorseypa1 - waiting to hear back from Chloe Brown about the additional card and dashboard.  I'll let you know as soon as I hear back.

Thanks,
Zack

**Comment by michelle-dooley** [comment]
@zlaw Chloe is no longer on the team or the contract for that matter.  What additional card and dashboard are you referring to outside of updating the one discussed above (landing pages by campaign) with the new campaign?

**Comment by zlaw** [comment]
Hi @michelle-dooley and @dorseypa1 - looks like there are 2 cards and 1 dashboard in the screen shot above.  Before removing them, I just wanted to be sure that no one was using them.  Since Chloe is no longer on the team, I think it's safe to remove all 3.

**Comment by zlaw** [comment]
Hey Michelle - as we discussed on the call, let's just remove the VA Notify - Test service and we're good to go!

**Comment by michelle-dooley** [comment]
Hey @zlaw - Updates have been completed.  The 22-5490 campaign has been added and test data has been removed.  Please take a look and let me know if everything looks right. 
https://va-gov.domo.com/page/2040841289

**Comment by michelle-dooley** [comment]
Closing this ticket as work is complete.  Please reach out if you have any questions or need anything else.
